vulp/actuation/BulletInterface.h

```diff
@@ -103,6 +103,9 @@ class BulletInterface : public Interface {
     //! If true, set gravity to -9.81 m/s².
     bool gravity = true;
 
+    //! If true, load a ground plane.
+    bool ground = true;
```

Comment:
This argument used to be called ``floor``. If we revert to that, let's call it the same for a relative extent of backward compatibility?

```
-  if (params.floor) {
-    bullet_.loadURDF(find_plane_urdf(params.argv0));
-  }
```

Check out ``git diff dd18e3aa803a92b603af6a0e458f06a6eff250d8`` for differences.